.github/workflows/build_and_test.yml

```diff
@@ -3,7 +3,7 @@ name: Build and test
 on:
   push:
     branches:
-    - branch-2.4
+    - '**'
```

Comment:
You'll need to revert this - you can put [branch-2.4] in the PR title if it's somehow not triggering

Reply:
Yes I will revert this, I temporarily did this so I can run actions in my own fork :) thanks